Added your_bank_account_id to request docs (#57)
splitpayments/api-documentation: source/index.html.md

|» amount|body|number|true|Amount in cents to pay the initiator|
|» authoriser_contact_id|body|string|true|The Contact the payment will be requested from (`Contact.data.id`)'|
|» precheck_funds|body|boolean|false|Enforce prechecking of available funds before approving the Payment Request. see [Payment Request - Precheck Funds](/#precheck-funds-lifecycle)|

> |» your_bank_account_id|body|string|false|Refers to `initiator_bank_account_id`. You can use this to identify issues related to the initiator's bank account.|

Reviewer: In the context of an API response, this statement is correct. Here we're posting a request. (Ditto others)
```suggestion
|» your_bank_account_id|body|string|false|Refers to `initiator_bank_account_id`. You can use this to initiate a transaction with an account other than your 'Primary' bank account.|
```

|» amount|body|number|true|Amount in cents to pay the initiator|
|» authoriser_contact_id|body|string|true|The Contact the payment will be requested from (`Contact.data.id`)'|
|» precheck_funds|body|boolean|false|Enforce prechecking of available funds before approving the Payment Request. see [Payment Request - Precheck Funds](/#precheck-funds-lifecycle)|

> |» your_bank_account_id|body|string|false|Specify which bank account we should use to settle/disburse the funds for this transaction. If you don't use this, your primary account will be used.|

Reviewer 2: ```suggestion
|» your_bank_account_id|body|string|false|Specify where we should settle the funds for this transaction. If omitted, your primary bank account will be used.|
```

|body|body|[MakeAPaymentRequest](#schemamakeapaymentrequest)|true|No description|
|» description|body|string|true|User description. Only visible to the payer|
|» matures_at|body|string|true|Date & time in UTC ISO8601 the Payment should be processed. (Can not be earlier than the start of current day)|

> |» your_bank_account_id|body|string|false|Specify which bank account we should use to settle/disburse the funds for this transaction. If you don't use this, your primary account will be used.|

Reviewer 2: ```suggestion
|» your_bank_account_id|body|string|false|Specify where we should take the funds this transaction. If omitted, your primary bank account will be used.|
```

Author: Nice 👍 I like this one.
